packages/playground/src/style.css

```diff
@@ -155,3 +155,7 @@ body {
   padding: 10px;
   margin: 20px;
 }
+
+.output-dropdown{
+  position: absolute;
```

Comment:
this still need a bit more tweaking, right now it shows up at the top left below the editor, try using `z-index` to raise it up and margin to make it show up on the right